.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tools/ocean_model_grid_generator"]
+	path = tools/ocean_model_grid_generator
+	url = https://github.com/NOAA-GFDL/ocean_model_grid_generator
```

Comment:
@ngs333 the ocean grid updates are expected in this PR due to the merge conflict resolution right?

Reply:
@ceblanton 
Yes. The ocean model grid generators (OMGG) are merged both to the baseline and to the branch of this PR.
There were merge conflicts between a previous PR branch of this work and the current baseline do to the OMGG work , but this branch of this PR includes  a fix of the conflicts.

Reply:
That's the thing! I thought that this diff should not show up since the old version (in the PR) is not the current state of the baseline (master)

Reply:
@ceblanton 
After the OMGG was merged to the baseline, the old version of the PR showed merge conflicts with the baseline.
I created a new branch that pulled both (merged them) and manually resolved the conflicts. This PR pulls that in.